app/models/transaction.rb

```diff
@@ -0,0 +1,7 @@
+class Transaction < ApplicationRecord
+  belongs_to :invoice
+
+  validates_presence_of :credit_card_number, :result, :invoice_id
+
+  enum result: {success: 0, failed: 1}
```

Comment:
Do we need to test these `enum` lines?

Reply:
That is a good question. I found this [stack overflow](https://stackoverflow.com/questions/29550958/how-should-i-test-rails-enums), so I think we may have to test this later.